cassandra/query.py

```diff
@@ -253,6 +253,12 @@ class Statement(object):
     .. versionadded:: 2.1.3
     """
 
+    table = None
+    """
+    The string name of the table this query acts on. This is used when 
+    :class`~.TokenAwarePolicy` is configured in the profile load balancing policy.
+    """
+
```

Comment:
I think it's only used when Tablets are used, considering that it's added in this PR. It should be stated in the comment.

Reply:
Added information about that